tests/test_densepose.py

```diff
@@ -106,7 +106,7 @@ def test_video(model, chimp_video_path, tmp_path):
         )
 
         # output to disk
-        assert anatomy_info.shape == (8, 46)
+        assert anatomy_info.shape == (10, 46)
```

Comment:
unrelated to caching, this is due to new densepose model release: https://github.com/facebookresearch/detectron2/tree/main/projects/DensePose#whats-new